emulator/configuration.py

```diff
@@ -0,0 +1,9 @@
+from .cubesat import CubeSat
```

Comment:
careful with relative imports (it can quickly become messy). would prefer absolute import for everything